src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
-body {
-  margin: 0;
+@import "normalize";
+
+.container {
+  color: #fff;
+  font-size: 100px;
+  background-color: black;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  &__block {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 300px;
+  }
+
+  &__block--1 {
+    background-color: rgba(255, 0, 0, 100%);
+  }
+
+  &__block--2 {
+    background-color: #f00c;
+  }
+
+  &__block--3 {
+    background-color: #f009;
+  }
+
+  &__block--4 {
+    background-color: #f006;
+  }
+
+  &__block--5 {
+    background-color: #f003;
+  }
+
+  &__block--6 {
+    background-color: #f000;
+  }
+
+  @media (min-width: 900px) {
+    .container {
+      grid-template-columns: repeat(3, 1fr);
+    }
+  }
+}
+
+@media (min-width: 600px) and (max-width: 900px) {
+  .container__block--3 {
+    grid-column: 2 / 3;
+    grid-row: 2/3;
+  }
```

Comment:
It's easier to add a property fro this media query like in " @media (min-width: 900px) "

Reply:
Thanks for review, Rostyslav! I did this to properly arrange the blocks when resizing the screen. To be a snake. Last time we used a order. I decided to try to use a grid somehow

Reply:
Ok. But have you tried to use this " grid-template-columns: repeat(2, 1fr)" for media query (min-width: 600px) ? I think it will work the same way, and code will be shorter.

Reply:
Thank you, Rostyslav. you're right!